build/common/Makefile.common.mk

```diff
@@ -7,7 +7,7 @@ CONTROLLER_GEN_VERSION := v0.16.3
 # https://github.com/kubernetes-sigs/kustomize/releases/latest
 KUSTOMIZE_VERSION := v5.4.3
 # https://github.com/golangci/golangci-lint/releases/latest
-GOLANGCI_VERSION := v1.52.2
+GOLANGCI_VERSION := v1.64.8
```

Comment:
Looks like `gci` and `gofumpt` don't have new versions, bu there were bumps to `GoSec` and Kustomize mentioned in the issue. Not to pile too much on, but are those bumps feasible here?

Reply:
Yes, that'd be my vote--thanks!